packages/storybook/package.json

```diff
@@ -53,6 +53,8 @@
     "@storybook/addon-essentials": "~6.5.7",
     "@storybook/addon-links": "~6.5.7",
     "@storybook/addon-storyshots": "~6.5.7",
+    "@storybook/builder-webpack5": "6.5.12",
```

Comment:
There are some other packages like @storybook/api are still staying in ~6.5.7, I guess we might want to align them together? (maybe make them all into ~6.5.12?)

Reply:
We probably should but as there's no errors I imagine it will fall into 'isn't broke won't fix it'. In saying that storybook7 is currently in development and on like alpha v42 so easiest to wait for that then do a big upgrade!